src/monocle_apptrace/wrap_common.py

```diff
@@ -190,11 +191,18 @@ async def allm_wrapper(tracer, to_wrap, wrapped, instance, args, kwargs):
     else:
         name = f"langchain.task.{instance.__class__.__name__}"
     with tracer.start_as_current_span(name) as span:
-        update_llm_endpoint(curr_span=span, instance=instance)
+        if 'haystack.components.retrievers' in to_wrap['package'] and 'haystack.retriever' in span.name:
```

Comment:
Is this code new or just showing new in diff?

Reply:
no this not new code just same same as llm_wrapper but  for allm_wrapper